{
    "valid": false,
    "stats": {
        "tasks": 3,
        "errors": 90,
        "warnings": 0,
        "seconds": 0.854
    },
    "warnings": [],
    "errors": [],
    "tasks": [
        {
            "name": "deployments",
            "type": "table",
            "valid": false,
            "place": "deployments.csv",
            "labels": [
                "deploymentID",
                "locationID",
                "locationName",
                "latitude",
                "longitude",
                "coordinateUncertainty",
                "deploymentStart",
                "deploymentEnd",
                "deploymentGroups",
                "deploymentTags",
                "deploymentComments",
                "setupBy",
                "cameraID",
                "cameraModel",
                "cameraDelay",
                "cameraDepth",
                "cameraHeight",
                "cameraTilt",
                "cameraHeading",
                "detectionDistance",
                "timestampIssues",
                "baitUse",
                "featureType",
                "habitat"
            ],
            "stats": {
                "errors": 17,
                "warnings": 0,
                "seconds": 0.061,
                "md5": "cbb526ab5444cbbcf87d31f390024b89",
                "sha256": "ae0e30ce1031aba7def522947a4c0854bbbd44f0c2b77e9f36b9e41e7c73a3eb",
                "bytes": 503,
                "fields": 24,
                "rows": 1
            },
            "warnings": [],
            "errors": [
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"deploymentGroups\" in field setupBy at position \"9\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "deploymentGroups",
                    "fieldName": "setupBy",
                    "fieldNumber": 9
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"deploymentTags\" in field cameraID at position \"10\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "deploymentTags",
                    "fieldName": "cameraID",
                    "fieldNumber": 10
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"deploymentComments\" in field cameraModel at position \"11\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "deploymentComments",
                    "fieldName": "cameraModel",
                    "fieldNumber": 11
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"setupBy\" in field cameraDelay at position \"12\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "setupBy",
                    "fieldName": "cameraDelay",
                    "fieldNumber": 12
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"cameraID\" in field cameraHeight at position \"13\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "cameraID",
                    "fieldName": "cameraHeight",
                    "fieldNumber": 13
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"cameraModel\" in field cameraDepth at position \"14\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "cameraModel",
                    "fieldName": "cameraDepth",
                    "fieldNumber": 14
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"cameraDelay\" in field cameraTilt at position \"15\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "cameraDelay",
                    "fieldName": "cameraTilt",
                    "fieldNumber": 15
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"cameraDepth\" in field cameraHeading at position \"16\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "cameraDepth",
                    "fieldName": "cameraHeading",
                    "fieldNumber": 16
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"cameraHeight\" in field detectionDistance at position \"17\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "cameraHeight",
                    "fieldName": "detectionDistance",
                    "fieldNumber": 17
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"cameraTilt\" in field timestampIssues at position \"18\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "cameraTilt",
                    "fieldName": "timestampIssues",
                    "fieldNumber": 18
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"cameraHeading\" in field baitUse at position \"19\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "cameraHeading",
                    "fieldName": "baitUse",
                    "fieldNumber": 19
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"detectionDistance\" in field featureType at position \"20\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "detectionDistance",
                    "fieldName": "featureType",
                    "fieldNumber": 20
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"timestampIssues\" in field habitat at position \"21\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "timestampIssues",
                    "fieldName": "habitat",
                    "fieldNumber": 21
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"baitUse\" in field deploymentGroups at position \"22\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "baitUse",
                    "fieldName": "deploymentGroups",
                    "fieldNumber": 22
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"featureType\" in field deploymentTags at position \"23\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "featureType",
                    "fieldName": "deploymentTags",
                    "fieldNumber": 23
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"habitat\" in field deploymentComments at position \"24\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "deploymentID",
                        "locationID",
                        "locationName",
                        "latitude",
                        "longitude",
                        "coordinateUncertainty",
                        "deploymentStart",
                        "deploymentEnd",
                        "deploymentGroups",
                        "deploymentTags",
                        "deploymentComments",
                        "setupBy",
                        "cameraID",
                        "cameraModel",
                        "cameraDelay",
                        "cameraDepth",
                        "cameraHeight",
                        "cameraTilt",
                        "cameraHeading",
                        "detectionDistance",
                        "timestampIssues",
                        "baitUse",
                        "featureType",
                        "habitat"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "habitat",
                    "fieldName": "deploymentComments",
                    "fieldNumber": 24
                },
                {
                    "type": "type-error",
                    "title": "Type Error",
                    "description": "The value does not match the schema type and format for this field.",
                    "message": "Type error in the cell \"SpyPoint Camera-LINK-MICRO-LTE\" in row \"2\" and field \"cameraDepth\" at position \"14\": type is \"number/default\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "type is \"number/default\"",
                    "cells": [
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "test-DEPLOYMENT-LOCATION",
                        "41.907144",
                        "-87.652254",
                        "",
                        "2024-02-25T00:41:29-0600",
                        "2024-02-25T10:08:41-0600",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "SpyPoint Camera-LINK-MICRO-LTE",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 2,
                    "cell": "SpyPoint Camera-LINK-MICRO-LTE",
                    "fieldName": "cameraDepth",
                    "fieldNumber": 14
                }
            ]
        },
        {
            "name": "media",
            "type": "table",
            "valid": true,
            "place": "media.csv",
            "labels": [
                "mediaID",
                "deploymentID",
                "captureMethod",
                "timestamp",
                "filePath",
                "filePublic",
                "fileName",
                "fileMediatype",
                "exifData",
                "favorite",
                "mediaComments"
            ],
            "stats": {
                "errors": 0,
                "warnings": 0,
                "seconds": 0.029,
                "md5": "09793db05d5adef740dee1a241e06e30",
                "sha256": "f9b31275010db06b10aedae1efb05d9f57533aa00f384c11db065d147705756e",
                "bytes": 28683,
                "fields": 11,
                "rows": 11
            },
            "warnings": [],
            "errors": []
        },
        {
            "name": "observations",
            "type": "table",
            "valid": false,
            "place": "observations.csv",
            "labels": [
                "observationID",
                "mediaID",
                "deploymentID",
                "eventStart",
                "eventEnd",
                "observationLevel",
                "observationType",
                "eventID",
                "cameraSetupType",
                "scientificName",
                "count",
                "lifeStage",
                "sex",
                "behavior",
                "individualID",
                "individualPositionRadius",
                "individualPositionAngle",
                "individualSpeed",
                "bboxX",
                "bboxY",
                "bboxWidth",
                "bboxHeight",
                "classificationMethod",
                "classifiedBy",
                "classificationTimestamp",
                "classificationProbability",
                "observationTags",
                "observationComments"
            ],
            "stats": {
                "errors": 73,
                "warnings": 0,
                "seconds": 0.046,
                "md5": "5ec5e841f78d21a44e6042a7120f0f82",
                "sha256": "94a4359bba0d1b053e98aaf4af7a155f50f1aa09a46edf0e31cc5d01d94b56c6",
                "bytes": 1399,
                "fields": 28,
                "rows": 11
            },
            "warnings": [],
            "errors": [
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"mediaID\" in field deploymentID at position \"2\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "observationID",
                        "mediaID",
                        "deploymentID",
                        "eventStart",
                        "eventEnd",
                        "observationLevel",
                        "observationType",
                        "eventID",
                        "cameraSetupType",
                        "scientificName",
                        "count",
                        "lifeStage",
                        "sex",
                        "behavior",
                        "individualID",
                        "individualPositionRadius",
                        "individualPositionAngle",
                        "individualSpeed",
                        "bboxX",
                        "bboxY",
                        "bboxWidth",
                        "bboxHeight",
                        "classificationMethod",
                        "classifiedBy",
                        "classificationTimestamp",
                        "classificationProbability",
                        "observationTags",
                        "observationComments"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "mediaID",
                    "fieldName": "deploymentID",
                    "fieldNumber": 2
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"deploymentID\" in field mediaID at position \"3\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "observationID",
                        "mediaID",
                        "deploymentID",
                        "eventStart",
                        "eventEnd",
                        "observationLevel",
                        "observationType",
                        "eventID",
                        "cameraSetupType",
                        "scientificName",
                        "count",
                        "lifeStage",
                        "sex",
                        "behavior",
                        "individualID",
                        "individualPositionRadius",
                        "individualPositionAngle",
                        "individualSpeed",
                        "bboxX",
                        "bboxY",
                        "bboxWidth",
                        "bboxHeight",
                        "classificationMethod",
                        "classifiedBy",
                        "classificationTimestamp",
                        "classificationProbability",
                        "observationTags",
                        "observationComments"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "deploymentID",
                    "fieldName": "mediaID",
                    "fieldNumber": 3
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"eventStart\" in field eventID at position \"4\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "observationID",
                        "mediaID",
                        "deploymentID",
                        "eventStart",
                        "eventEnd",
                        "observationLevel",
                        "observationType",
                        "eventID",
                        "cameraSetupType",
                        "scientificName",
                        "count",
                        "lifeStage",
                        "sex",
                        "behavior",
                        "individualID",
                        "individualPositionRadius",
                        "individualPositionAngle",
                        "individualSpeed",
                        "bboxX",
                        "bboxY",
                        "bboxWidth",
                        "bboxHeight",
                        "classificationMethod",
                        "classifiedBy",
                        "classificationTimestamp",
                        "classificationProbability",
                        "observationTags",
                        "observationComments"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "eventStart",
                    "fieldName": "eventID",
                    "fieldNumber": 4
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"eventEnd\" in field eventStart at position \"5\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "observationID",
                        "mediaID",
                        "deploymentID",
                        "eventStart",
                        "eventEnd",
                        "observationLevel",
                        "observationType",
                        "eventID",
                        "cameraSetupType",
                        "scientificName",
                        "count",
                        "lifeStage",
                        "sex",
                        "behavior",
                        "individualID",
                        "individualPositionRadius",
                        "individualPositionAngle",
                        "individualSpeed",
                        "bboxX",
                        "bboxY",
                        "bboxWidth",
                        "bboxHeight",
                        "classificationMethod",
                        "classifiedBy",
                        "classificationTimestamp",
                        "classificationProbability",
                        "observationTags",
                        "observationComments"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "eventEnd",
                    "fieldName": "eventStart",
                    "fieldNumber": 5
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"observationLevel\" in field eventEnd at position \"6\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "observationID",
                        "mediaID",
                        "deploymentID",
                        "eventStart",
                        "eventEnd",
                        "observationLevel",
                        "observationType",
                        "eventID",
                        "cameraSetupType",
                        "scientificName",
                        "count",
                        "lifeStage",
                        "sex",
                        "behavior",
                        "individualID",
                        "individualPositionRadius",
                        "individualPositionAngle",
                        "individualSpeed",
                        "bboxX",
                        "bboxY",
                        "bboxWidth",
                        "bboxHeight",
                        "classificationMethod",
                        "classifiedBy",
                        "classificationTimestamp",
                        "classificationProbability",
                        "observationTags",
                        "observationComments"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "observationLevel",
                    "fieldName": "eventEnd",
                    "fieldNumber": 6
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"observationType\" in field observationLevel at position \"7\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "observationID",
                        "mediaID",
                        "deploymentID",
                        "eventStart",
                        "eventEnd",
                        "observationLevel",
                        "observationType",
                        "eventID",
                        "cameraSetupType",
                        "scientificName",
                        "count",
                        "lifeStage",
                        "sex",
                        "behavior",
                        "individualID",
                        "individualPositionRadius",
                        "individualPositionAngle",
                        "individualSpeed",
                        "bboxX",
                        "bboxY",
                        "bboxWidth",
                        "bboxHeight",
                        "classificationMethod",
                        "classifiedBy",
                        "classificationTimestamp",
                        "classificationProbability",
                        "observationTags",
                        "observationComments"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "observationType",
                    "fieldName": "observationLevel",
                    "fieldNumber": 7
                },
                {
                    "type": "incorrect-label",
                    "title": "Incorrect Label",
                    "description": "One of the data source header does not match the field name defined in the schema.",
                    "message": "Label \"eventID\" in field observationType at position \"8\" does not match the field name in the schema",
                    "tags": [
                        "#table",
                        "#header",
                        "#label"
                    ],
                    "note": "",
                    "labels": [
                        "observationID",
                        "mediaID",
                        "deploymentID",
                        "eventStart",
                        "eventEnd",
                        "observationLevel",
                        "observationType",
                        "eventID",
                        "cameraSetupType",
                        "scientificName",
                        "count",
                        "lifeStage",
                        "sex",
                        "behavior",
                        "individualID",
                        "individualPositionRadius",
                        "individualPositionAngle",
                        "individualSpeed",
                        "bboxX",
                        "bboxY",
                        "bboxWidth",
                        "bboxHeight",
                        "classificationMethod",
                        "classifiedBy",
                        "classificationTimestamp",
                        "classificationProbability",
                        "observationTags",
                        "observationComments"
                    ],
                    "rowNumbers": [
                        1
                    ],
                    "label": "eventID",
                    "fieldName": "observationType",
                    "fieldNumber": 8
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"2\" and field \"eventStart\" at position \"5\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0094_1",
                        "PICT0094",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 2,
                    "cell": "",
                    "fieldName": "eventStart",
                    "fieldNumber": 5
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"2\" and field \"eventEnd\" at position \"6\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0094_1",
                        "PICT0094",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 2,
                    "cell": "",
                    "fieldName": "eventEnd",
                    "fieldNumber": 6
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"2\" and field \"observationLevel\" at position \"7\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0094_1",
                        "PICT0094",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 2,
                    "cell": "",
                    "fieldName": "observationLevel",
                    "fieldNumber": 7
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"2\" and field \"observationType\" at position \"8\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0094_1",
                        "PICT0094",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 2,
                    "cell": "",
                    "fieldName": "observationType",
                    "fieldNumber": 8
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"2\" violates the foreign key: for \"deploymentID\": values \"PICT0094\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"deploymentID\": values \"PICT0094\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "cells": [
                        "PICT0094_1",
                        "PICT0094",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 2,
                    "fieldNames": [
                        "deploymentID"
                    ],
                    "fieldCells": [
                        "PICT0094"
                    ],
                    "referenceName": "deployments",
                    "referenceFieldNames": [
                        "deploymentID"
                    ]
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"2\" violates the foreign key: for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "cells": [
                        "PICT0094_1",
                        "PICT0094",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 2,
                    "fieldNames": [
                        "mediaID"
                    ],
                    "fieldCells": [
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA"
                    ],
                    "referenceName": "media",
                    "referenceFieldNames": [
                        "mediaID"
                    ]
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"3\" and field \"eventStart\" at position \"5\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0095_1",
                        "PICT0095",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 3,
                    "cell": "",
                    "fieldName": "eventStart",
                    "fieldNumber": 5
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"3\" and field \"eventEnd\" at position \"6\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0095_1",
                        "PICT0095",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 3,
                    "cell": "",
                    "fieldName": "eventEnd",
                    "fieldNumber": 6
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"3\" and field \"observationLevel\" at position \"7\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0095_1",
                        "PICT0095",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 3,
                    "cell": "",
                    "fieldName": "observationLevel",
                    "fieldNumber": 7
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"3\" and field \"observationType\" at position \"8\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0095_1",
                        "PICT0095",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 3,
                    "cell": "",
                    "fieldName": "observationType",
                    "fieldNumber": 8
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"3\" violates the foreign key: for \"deploymentID\": values \"PICT0095\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"deploymentID\": values \"PICT0095\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "cells": [
                        "PICT0095_1",
                        "PICT0095",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 3,
                    "fieldNames": [
                        "deploymentID"
                    ],
                    "fieldCells": [
                        "PICT0095"
                    ],
                    "referenceName": "deployments",
                    "referenceFieldNames": [
                        "deploymentID"
                    ]
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"3\" violates the foreign key: for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "cells": [
                        "PICT0095_1",
                        "PICT0095",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 3,
                    "fieldNames": [
                        "mediaID"
                    ],
                    "fieldCells": [
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA"
                    ],
                    "referenceName": "media",
                    "referenceFieldNames": [
                        "mediaID"
                    ]
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"4\" and field \"eventStart\" at position \"5\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0097_1",
                        "PICT0097",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 4,
                    "cell": "",
                    "fieldName": "eventStart",
                    "fieldNumber": 5
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"4\" and field \"eventEnd\" at position \"6\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0097_1",
                        "PICT0097",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 4,
                    "cell": "",
                    "fieldName": "eventEnd",
                    "fieldNumber": 6
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"4\" and field \"observationLevel\" at position \"7\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0097_1",
                        "PICT0097",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 4,
                    "cell": "",
                    "fieldName": "observationLevel",
                    "fieldNumber": 7
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"4\" and field \"observationType\" at position \"8\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0097_1",
                        "PICT0097",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 4,
                    "cell": "",
                    "fieldName": "observationType",
                    "fieldNumber": 8
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"4\" violates the foreign key: for \"deploymentID\": values \"PICT0097\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"deploymentID\": values \"PICT0097\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "cells": [
                        "PICT0097_1",
                        "PICT0097",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 4,
                    "fieldNames": [
                        "deploymentID"
                    ],
                    "fieldCells": [
                        "PICT0097"
                    ],
                    "referenceName": "deployments",
                    "referenceFieldNames": [
                        "deploymentID"
                    ]
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"4\" violates the foreign key: for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "cells": [
                        "PICT0097_1",
                        "PICT0097",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 4,
                    "fieldNames": [
                        "mediaID"
                    ],
                    "fieldCells": [
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA"
                    ],
                    "referenceName": "media",
                    "referenceFieldNames": [
                        "mediaID"
                    ]
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"5\" and field \"eventStart\" at position \"5\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0096_1",
                        "PICT0096",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 5,
                    "cell": "",
                    "fieldName": "eventStart",
                    "fieldNumber": 5
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"5\" and field \"eventEnd\" at position \"6\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0096_1",
                        "PICT0096",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 5,
                    "cell": "",
                    "fieldName": "eventEnd",
                    "fieldNumber": 6
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"5\" and field \"observationLevel\" at position \"7\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0096_1",
                        "PICT0096",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 5,
                    "cell": "",
                    "fieldName": "observationLevel",
                    "fieldNumber": 7
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"5\" and field \"observationType\" at position \"8\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0096_1",
                        "PICT0096",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 5,
                    "cell": "",
                    "fieldName": "observationType",
                    "fieldNumber": 8
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"5\" violates the foreign key: for \"deploymentID\": values \"PICT0096\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"deploymentID\": values \"PICT0096\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "cells": [
                        "PICT0096_1",
                        "PICT0096",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 5,
                    "fieldNames": [
                        "deploymentID"
                    ],
                    "fieldCells": [
                        "PICT0096"
                    ],
                    "referenceName": "deployments",
                    "referenceFieldNames": [
                        "deploymentID"
                    ]
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"5\" violates the foreign key: for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "cells": [
                        "PICT0096_1",
                        "PICT0096",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 5,
                    "fieldNames": [
                        "mediaID"
                    ],
                    "fieldCells": [
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA"
                    ],
                    "referenceName": "media",
                    "referenceFieldNames": [
                        "mediaID"
                    ]
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"6\" and field \"eventStart\" at position \"5\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0092_1",
                        "PICT0092",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 6,
                    "cell": "",
                    "fieldName": "eventStart",
                    "fieldNumber": 5
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"6\" and field \"eventEnd\" at position \"6\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0092_1",
                        "PICT0092",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 6,
                    "cell": "",
                    "fieldName": "eventEnd",
                    "fieldNumber": 6
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"6\" and field \"observationLevel\" at position \"7\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0092_1",
                        "PICT0092",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 6,
                    "cell": "",
                    "fieldName": "observationLevel",
                    "fieldNumber": 7
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"6\" and field \"observationType\" at position \"8\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0092_1",
                        "PICT0092",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 6,
                    "cell": "",
                    "fieldName": "observationType",
                    "fieldNumber": 8
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"6\" violates the foreign key: for \"deploymentID\": values \"PICT0092\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"deploymentID\": values \"PICT0092\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "cells": [
                        "PICT0092_1",
                        "PICT0092",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 6,
                    "fieldNames": [
                        "deploymentID"
                    ],
                    "fieldCells": [
                        "PICT0092"
                    ],
                    "referenceName": "deployments",
                    "referenceFieldNames": [
                        "deploymentID"
                    ]
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"6\" violates the foreign key: for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "cells": [
                        "PICT0092_1",
                        "PICT0092",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 6,
                    "fieldNames": [
                        "mediaID"
                    ],
                    "fieldCells": [
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA"
                    ],
                    "referenceName": "media",
                    "referenceFieldNames": [
                        "mediaID"
                    ]
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"7\" and field \"eventStart\" at position \"5\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0093_1",
                        "PICT0093",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 7,
                    "cell": "",
                    "fieldName": "eventStart",
                    "fieldNumber": 5
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"7\" and field \"eventEnd\" at position \"6\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0093_1",
                        "PICT0093",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 7,
                    "cell": "",
                    "fieldName": "eventEnd",
                    "fieldNumber": 6
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"7\" and field \"observationLevel\" at position \"7\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0093_1",
                        "PICT0093",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 7,
                    "cell": "",
                    "fieldName": "observationLevel",
                    "fieldNumber": 7
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"7\" and field \"observationType\" at position \"8\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0093_1",
                        "PICT0093",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 7,
                    "cell": "",
                    "fieldName": "observationType",
                    "fieldNumber": 8
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"7\" violates the foreign key: for \"deploymentID\": values \"PICT0093\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"deploymentID\": values \"PICT0093\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "cells": [
                        "PICT0093_1",
                        "PICT0093",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 7,
                    "fieldNames": [
                        "deploymentID"
                    ],
                    "fieldCells": [
                        "PICT0093"
                    ],
                    "referenceName": "deployments",
                    "referenceFieldNames": [
                        "deploymentID"
                    ]
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"7\" violates the foreign key: for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "cells": [
                        "PICT0093_1",
                        "PICT0093",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 7,
                    "fieldNames": [
                        "mediaID"
                    ],
                    "fieldCells": [
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA"
                    ],
                    "referenceName": "media",
                    "referenceFieldNames": [
                        "mediaID"
                    ]
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"8\" and field \"eventStart\" at position \"5\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0091_1",
                        "PICT0091",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 8,
                    "cell": "",
                    "fieldName": "eventStart",
                    "fieldNumber": 5
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"8\" and field \"eventEnd\" at position \"6\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0091_1",
                        "PICT0091",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 8,
                    "cell": "",
                    "fieldName": "eventEnd",
                    "fieldNumber": 6
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"8\" and field \"observationLevel\" at position \"7\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0091_1",
                        "PICT0091",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 8,
                    "cell": "",
                    "fieldName": "observationLevel",
                    "fieldNumber": 7
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"8\" and field \"observationType\" at position \"8\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0091_1",
                        "PICT0091",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 8,
                    "cell": "",
                    "fieldName": "observationType",
                    "fieldNumber": 8
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"8\" violates the foreign key: for \"deploymentID\": values \"PICT0091\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"deploymentID\": values \"PICT0091\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "cells": [
                        "PICT0091_1",
                        "PICT0091",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 8,
                    "fieldNames": [
                        "deploymentID"
                    ],
                    "fieldCells": [
                        "PICT0091"
                    ],
                    "referenceName": "deployments",
                    "referenceFieldNames": [
                        "deploymentID"
                    ]
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"8\" violates the foreign key: for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "cells": [
                        "PICT0091_1",
                        "PICT0091",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 8,
                    "fieldNames": [
                        "mediaID"
                    ],
                    "fieldCells": [
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA"
                    ],
                    "referenceName": "media",
                    "referenceFieldNames": [
                        "mediaID"
                    ]
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"9\" and field \"eventStart\" at position \"5\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0090_1",
                        "PICT0090",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 9,
                    "cell": "",
                    "fieldName": "eventStart",
                    "fieldNumber": 5
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"9\" and field \"eventEnd\" at position \"6\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0090_1",
                        "PICT0090",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 9,
                    "cell": "",
                    "fieldName": "eventEnd",
                    "fieldNumber": 6
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"9\" and field \"observationLevel\" at position \"7\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0090_1",
                        "PICT0090",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 9,
                    "cell": "",
                    "fieldName": "observationLevel",
                    "fieldNumber": 7
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"9\" and field \"observationType\" at position \"8\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0090_1",
                        "PICT0090",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 9,
                    "cell": "",
                    "fieldName": "observationType",
                    "fieldNumber": 8
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"9\" violates the foreign key: for \"deploymentID\": values \"PICT0090\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"deploymentID\": values \"PICT0090\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "cells": [
                        "PICT0090_1",
                        "PICT0090",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 9,
                    "fieldNames": [
                        "deploymentID"
                    ],
                    "fieldCells": [
                        "PICT0090"
                    ],
                    "referenceName": "deployments",
                    "referenceFieldNames": [
                        "deploymentID"
                    ]
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"9\" violates the foreign key: for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "cells": [
                        "PICT0090_1",
                        "PICT0090",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 9,
                    "fieldNames": [
                        "mediaID"
                    ],
                    "fieldCells": [
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA"
                    ],
                    "referenceName": "media",
                    "referenceFieldNames": [
                        "mediaID"
                    ]
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"10\" and field \"eventStart\" at position \"5\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0100_1",
                        "PICT0100",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 10,
                    "cell": "",
                    "fieldName": "eventStart",
                    "fieldNumber": 5
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"10\" and field \"eventEnd\" at position \"6\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0100_1",
                        "PICT0100",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 10,
                    "cell": "",
                    "fieldName": "eventEnd",
                    "fieldNumber": 6
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"10\" and field \"observationLevel\" at position \"7\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0100_1",
                        "PICT0100",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 10,
                    "cell": "",
                    "fieldName": "observationLevel",
                    "fieldNumber": 7
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"10\" and field \"observationType\" at position \"8\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0100_1",
                        "PICT0100",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 10,
                    "cell": "",
                    "fieldName": "observationType",
                    "fieldNumber": 8
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"10\" violates the foreign key: for \"deploymentID\": values \"PICT0100\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"deploymentID\": values \"PICT0100\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "cells": [
                        "PICT0100_1",
                        "PICT0100",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 10,
                    "fieldNames": [
                        "deploymentID"
                    ],
                    "fieldCells": [
                        "PICT0100"
                    ],
                    "referenceName": "deployments",
                    "referenceFieldNames": [
                        "deploymentID"
                    ]
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"10\" violates the foreign key: for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "cells": [
                        "PICT0100_1",
                        "PICT0100",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 10,
                    "fieldNames": [
                        "mediaID"
                    ],
                    "fieldCells": [
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA"
                    ],
                    "referenceName": "media",
                    "referenceFieldNames": [
                        "mediaID"
                    ]
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"11\" and field \"eventStart\" at position \"5\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0098_1",
                        "PICT0098",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 11,
                    "cell": "",
                    "fieldName": "eventStart",
                    "fieldNumber": 5
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"11\" and field \"eventEnd\" at position \"6\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0098_1",
                        "PICT0098",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 11,
                    "cell": "",
                    "fieldName": "eventEnd",
                    "fieldNumber": 6
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"11\" and field \"observationLevel\" at position \"7\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0098_1",
                        "PICT0098",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 11,
                    "cell": "",
                    "fieldName": "observationLevel",
                    "fieldNumber": 7
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"11\" and field \"observationType\" at position \"8\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0098_1",
                        "PICT0098",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 11,
                    "cell": "",
                    "fieldName": "observationType",
                    "fieldNumber": 8
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"11\" violates the foreign key: for \"deploymentID\": values \"PICT0098\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"deploymentID\": values \"PICT0098\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "cells": [
                        "PICT0098_1",
                        "PICT0098",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 11,
                    "fieldNames": [
                        "deploymentID"
                    ],
                    "fieldCells": [
                        "PICT0098"
                    ],
                    "referenceName": "deployments",
                    "referenceFieldNames": [
                        "deploymentID"
                    ]
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"11\" violates the foreign key: for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "cells": [
                        "PICT0098_1",
                        "PICT0098",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 11,
                    "fieldNames": [
                        "mediaID"
                    ],
                    "fieldCells": [
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA"
                    ],
                    "referenceName": "media",
                    "referenceFieldNames": [
                        "mediaID"
                    ]
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"12\" and field \"eventStart\" at position \"5\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0099_1",
                        "PICT0099",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 12,
                    "cell": "",
                    "fieldName": "eventStart",
                    "fieldNumber": 5
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"12\" and field \"eventEnd\" at position \"6\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0099_1",
                        "PICT0099",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 12,
                    "cell": "",
                    "fieldName": "eventEnd",
                    "fieldNumber": 6
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"12\" and field \"observationLevel\" at position \"7\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0099_1",
                        "PICT0099",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 12,
                    "cell": "",
                    "fieldName": "observationLevel",
                    "fieldNumber": 7
                },
                {
                    "type": "constraint-error",
                    "title": "Constraint Error",
                    "description": "A field value does not conform to a constraint.",
                    "message": "The cell \"\" in row at position \"12\" and field \"observationType\" at position \"8\" does not conform to a constraint: constraint \"required\" is \"True\"",
                    "tags": [
                        "#table",
                        "#row",
                        "#cell"
                    ],
                    "note": "constraint \"required\" is \"True\"",
                    "cells": [
                        "PICT0099_1",
                        "PICT0099",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 12,
                    "cell": "",
                    "fieldName": "observationType",
                    "fieldNumber": 8
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"12\" violates the foreign key: for \"deploymentID\": values \"PICT0099\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"deploymentID\": values \"PICT0099\" not found in the lookup table \"deployments\" as \"deploymentID\"",
                    "cells": [
                        "PICT0099_1",
                        "PICT0099",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 12,
                    "fieldNames": [
                        "deploymentID"
                    ],
                    "fieldCells": [
                        "PICT0099"
                    ],
                    "referenceName": "deployments",
                    "referenceFieldNames": [
                        "deploymentID"
                    ]
                },
                {
                    "type": "foreign-key",
                    "title": "ForeignKey Error",
                    "description": "Values in the foreign key fields should exist in the reference table",
                    "message": "Row at position \"12\" violates the foreign key: for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "tags": [
                        "#table",
                        "#row"
                    ],
                    "note": "for \"mediaID\": values \"test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA\" not found in the lookup table \"media\" as \"mediaID\"",
                    "cells": [
                        "PICT0099_1",
                        "PICT0099",
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        "",
                        ""
                    ],
                    "rowNumber": 12,
                    "fieldNames": [
                        "mediaID"
                    ],
                    "fieldCells": [
                        "test-DEPLOYMENT-LOCATION-test-DATE-test-CAMERA"
                    ],
                    "referenceName": "media",
                    "referenceFieldNames": [
                        "mediaID"
                    ]
                }
            ]
        }
    ]
}